lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -97,6 +97,15 @@ bool firrtl::isDuplexValue(Value val) {
 
 /// Return the kind of port this is given the port type from a 'mem' decl.
 static MemOp::PortKind getMemPortKindFromType(FIRRTLType type) {
+  constexpr unsigned int addr = 1;
+  constexpr unsigned int en = 2;
+  constexpr unsigned int clk = 4;
+  constexpr unsigned int data = 8;
+  constexpr unsigned int mask = 16;
+  constexpr unsigned int rdata = 32;
+  constexpr unsigned int wdata = 64;
+  constexpr unsigned int wmask = 128;
+  constexpr unsigned int wmode = 256;
```

Comment:
If this gets bigger, I suggest using a style of `1 << 0`, `1 << 1`, etc., as it's easier to be sure everything is right at a glance and makes intent clear.